### ISSUE TITLE
Is "text-align: center" really deprecated and non-standard?!?

### ISSUE THREAD

**Issue by Hexstream** [mdn/mdn#529]
### Proposal

I noticed that [the MDN page for text-align](https://developer.mozilla.org/en-US/docs/Web/CSS/text-align) claims that `text-align: left`, `text-align: right` and `text-align: center` are all _deprecated and non-standard_.

I can understand in the case of `text-align: left` and `text-align: right`, since `text-align: start` and `text-align: end` seem like better alternatives anyway, but **I think `text-align: center` is obviously the best option in many cases, with no universal replacement.** Even that page itself uses `text-align: center` as an example, and I couldn't find any authoritative source explaining why `text-align: center` is or should be deprecated.

I think this is a bug in the MDN documentation and `text-align: center` is not in any way deprecated or non-standard.

**Comment by francescorn** [comment]
I agree, `text-align: center` is in the latest editor's draft: [css-text-4](https://drafts.csswg.org/css-text-4/#justification).

**Comment by Hexstream** [comment]
> In: [css-writing-modes-4](https://drafts.csswg.org/css-writing-modes-4/#direction) it says left to right features are not recommended because HTML UAs can turn off CSS styling. I think text-align: left and text-align: right are included.

I think you are misinterpreting the note, which just says you should use the html `dir` attribute rather than change the `direction` property directly with CSS.

**Comment by francescorn** [comment]
Thank you @Hexstream

**Comment by caugner** [comment]
Thanks for flagging, I will move this to the content repo, as the deprecated/non-standard badges are declared there:

https://github.com/mdn/content/blob/7889bb920fca5023a5c377627b945c41997f6f22/files/en-us/web/css/text-align/index.md#L51-L56
